flixel/input/mouse/FlxMouseEventManager.hx

```diff
@@ -443,22 +443,30 @@ class FlxMouseEventManager extends FlxBasic
 		for (camera in Register.object.cameras)
 		{
 			#if FLX_MOUSE
-			_point = FlxG.mouse.getWorldPosition(camera, _point);
-			
-			if (checkOverlapWithPoint(Register, _point, camera))
+			_point = FlxG.mouse.getScreenPosition(camera, _point);
```

Comment:
You named the function wrong! The function in `FlxPointer` is not the same name as in `FlxMEM`.